shared/helpers/puppeteer.js

```diff
@@ -128,6 +128,10 @@ async function render(page, componentName, renderOptions, browserOptions) {
   // property, which means we get a mangled value). As a workaround, we can
   // gather and `return` the values we need from inside the browser
   try {
+    if (!page.isJavaScriptEnabled()) {
```

Comment:
Another nice time saver!

Reply:
Kinda, it crashes trying to run the JS without JS otherwise 🤦‍♂️